examples/30_extended/create_upload_tutorial.py

```diff
@@ -286,12 +286,16 @@
 # Dataset is a pandas sparse dataframe
```

Comment:
could you please update this header to reflect the fact that pandas doesn't have this class any more.